### ISSUE TITLE
pycsw requires newer OWSLib version

### ISSUE THREAD

**Issue by tpo** [geopython/pycsw#652]
# Description

When I access the current pycsw I get:

```
Traceback (most recent call last):
  File "/usr/share/passenger/helper-scripts/wsgi-loader.py", line 325, in &lt;module&gt;
    app_module = load_app()
  File "/usr/share/passenger/helper-scripts/wsgi-loader.py", line 62, in load_app
    return imp.load_source('passenger_wsgi', startup_file)
  File "/usr/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/usr/local/lib/python3.5/dist-packages/pycsw-2.7.dev0-py3.5.egg/pycsw/wsgi.py", line 65, in <module>
    from pycsw import server
  File "/usr/local/lib/python3.5/dist-packages/pycsw-2.7.dev0-py3.5.egg/pycsw/server.py", line 49, in <module>
    from pycsw.ogc.csw import csw2, csw3
  File "/usr/local/lib/python3.5/dist-packages/pycsw-2.7.dev0-py3.5.egg/pycsw/ogc/csw/csw2.py", line 43, in <module>
    from pycsw.core import config, log, metadata, util
  File "/usr/local/lib/python3.5/dist-packages/pycsw-2.7.dev0-py3.5.egg/pycsw/core/metadata.py", line 40, in <module>
    from owslib.iso import MD_ImageDescription
ImportError: cannot import name 'MD_ImageDescription'
```

I have oswlib 0.19.1 installed (that is because oswlib > 0.19.1 depends on pyproj >=2 depends on libproj >=2 which is only available from Ubuntu 20.04 to which I am currently unable to upgrade to).

Note: I currently depend on pycsw because of a fix that was recently made.

It seems that the `MD_ImageDescription` mentioned above in the stack trace was introduced here: https://github.com/geopython/OWSLib/pull/726 so I guess pycsw should depend on a owslib version that includes that patch, i.e. owslib >= 0.22.0

# Environment

- operating system: Ubuntu 16.04
- Python version: 3.5
- pycsw version: HEAD as of 2021-02-05
- source/distribution
  - [x] git clone
  - [ ] DebianGIS/UbuntuGIS
  - [ ] PyPI
  - [ ] zip/tar.gz
  - [ ] other (please specify):
- web server
  - [x] Apache/mod_wsgi
  - [ ] CGI
  - [ ] other (please specify):

# Steps to Reproduce

# Additional Information

**Comment by tomkralidis** [comment]
This should be fixed given the OWSLib 0.24 release yesterday.  Can you try/validate and report back?

**Comment by tpo** [comment]
Hi @tomkralidis 

thanks a lot for having an eye on this ticket. 

Having trouble fitting all the moving pieces together I've switched to using the docker image of pycsw. At this time I can't see when I'll be able to try/validate whether the dependencies problem has been fixed. So unless you want to leave this ticket open I suggest to assume that the dependencies problem is fixed now and to close the ticket?

Thanks a lot & greetings,
*t
